modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -737,13 +737,15 @@ public void testAddObjectAction() throws Exception {
 			Assert.assertEquals(0, _argumentsList.size());
 
 			objectEntry = _objectEntryLocalService.partialUpdateObjectEntry(
-				TestPropsValues.getUserId(), objectEntry.getObjectEntryId(),
+				objectEntry.getObjectEntryId(), TestPropsValues.getUserId(),
+				ObjectEntryFolderConstants.
+					PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT,
+				ServiceContextTestUtil.getServiceContext(),
```

Comment:
you can use `objectEntry.getObjectEntryFolderId()` here

Reply:
Can you review the usages and apply:

- If the method is using objectEntry (like `objectEntry.getObjectEntryId()`), then use objectEntry.getObjectEntryFolderId().
- If the method is not using it, then you use `ObjectEntryFolderConstants.PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT`